markups/restructuredtext.py

```diff
@@ -15,7 +15,7 @@
 else:
 	class CustomHTMLTranslator(HTMLTranslator):
 		def starttag(self, node, tagname, suffix='\n', empty=False, **attributes):
-			if node.line is not None:
+			if node is not None and getattr(node, 'line', None) is not None:
```

Comment:
Actually, `node` cannot be None. In your case, it is dict. The `starttag` function from the original HTMLTranslator also assumes that, e.g. it uses `node.get('classes', [])`. So this can be further simplified as:

```suggestion
			if getattr(node, 'line', None) is not None:
```